### ISSUE TITLE
dead submit button on milestone section

### ISSUE THREAD

**Issue by csell5** [ThatConference/that-website#319]
the home page has a dead link. It could be coming from the api, or not even wired up.

![image](https://user-images.githubusercontent.com/772569/72915375-cc4dc100-3d05-11ea-9e75-96141ec54758.png)

**Comment by brettski** [comment]
This is more complicated than I assumed on the surface.  
This link button is controlled from the database in the notifications sub-collection of events. 

The button uses next/link object and we are sending it an entire URL instead of the page/path. 
The page/path ('wi/call-for-counselors') cannot be set as GraphQL is set to have that value as a URL. To set the path only, breaks this condition. 

The database has been set with the href of: `https://www.thatconference.com/wi/call-for-counselors`

The button still doesn't direct the user to the Call for Counselors page. I believe this is due to the fact that it is the same domain and should be a page reference, not a full URL reference.

I think the correct fix here is in GraphQL to store the path (string) not a URL. Looking for a second on this:

@csell5 @kburnell

**Comment by brettski** [comment]
Based on this thread: https://github.com/zeit/next.js/issues/8555
The suggestion is to replace the Link component with an <a> as Link will be removed later. 
Currently all pages are SSR so using Link really has no benefit when it comes to refeshing the page. Using Link or <a> will both cause the page to refresh as /call-for-counselors is a server side page.

**Comment by mcookWI** [comment]
@brettski I'm confused with this - when I click on the button, I am taken to the /wi/call-for-counselors page. So this really isn't a 'dead' button, or has something changed since your last comment?

**Comment by brettski** [comment]
@mcookWI There is a hack in place currently so it works. This hack changes the referenced link
from: `https://www.thatconference.com/wi/call-for-counselors`
to: `http://www.thatconference.com/wi/call-for-counselors`

The assumption on the hack is that Link sees the the link as external and redirects to it. If I put it back to the `From` you can reproduce the issue. 

The correction here is our approach to managing the link. We can't make it path as I suggested earlier as this is a notification link which could be used in any medium, not only the web site. 

I am pretty sure Sara had a suggestion for it.

**Comment by mcookWI** [comment]
@saragibby Do you recall the suggestion for this?

**Comment by saragibby** [comment]
Yeah, I don't recall anything about this one... but let's pause for a moment... February 7th, 2019... we were with people, sitting in coffee shops, complaining about loud restaurants... ahhh, those were the days.

So seeing as how this was essentially 3 years ago, what do we need to do for this button in the important dates section?
